docs/devel/hotswap.md

```diff
@@ -89,25 +69,13 @@ YYYY-MM-DD HH:MM:SS UTC	ERR	root  	karydia-dev	procs never ended 	CLOSE_WRITE,CL
 ```
 Issue & solution: __waiting timeout issue occured - try again or try to increase timeouts manually__
 
-1. add script parameter `-t` with a desired increase value, e.g. `5`, at `manifests-dev/deployment-dev.yml`
-```
-[...]
-        command:
-        - hotswap-dev
-+       - -t5
-        - -r
-[...]
-```
-2. deploy changes and try `make deploy-dev` again
-```
-kubectl apply -f manifests-dev/deployment-dev.yml
-```
+1. add script parameter `-t` with a desired increase value, e.g. `5` by using `helm install  --set dev.active=true --set dev.timeout=true --set dev.timeoutValue=5`.
```

Comment:
Is it possible to remove `--set dev.timeout=true` and instead just go with `--set dev.timeoutValue=5` only?
Furthermore, this is just an increase value. Maybe `--set dev.timeoutIncreaseValue=5` fits better!?

Reply:
Is it possible to run `helm install` multiple times (e.g. twice)?

Reply:
> Is it possible to run `helm install` multiple times (e.g. twice)?

Not when another implementation is already in place. However, `helm upgrade` updates an installation according to the changes specified.

Reply:
> Is it possible to remove `--set dev.timeout=true` and instead just go with `--set dev.timeoutValue=5` only?
> Furthermore, this is just an increase value. Maybe `--set dev.timeoutIncreaseValue=5` fits better!?

Done.